platforms/chibios/_util.h

```diff
@@ -2,6 +2,12 @@
 // SPDX-License-Identifier: GPL-2.0-or-later
 #pragma once
 
+#define RESIDENT_IN_RAM(funcname) __attribute__((section(".ram0_init." #funcname), noinline)) funcname
+
 #ifndef CORTEX_ENABLE_WFI_IDLE
 #    define CORTEX_ENABLE_WFI_IDLE TRUE
 #endif // CORTEX_ENABLE_WFI_IDLE
```

Comment:
This will need to go back into a `config.h` somewhere.

Reply:
Seems `platforms/chibios/config.h` is indeed included, so I've put just this define back in place.